presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,6 +171,55 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema. The user is granted
+the privileges based on the first matching rule read from top to bottom. If no
+rule matches, the user is not considered an owner of the schema. Each rule is
+composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``owner`` (required): boolean indicating whether the user is to be considered
+  an owner of the schema. Defaults to ``false``.
+
+For example, to provide ownership of all schemas to user ``admin``, treat all
+users as owners of ``default`` schema and prevent user ``guest`` from ownership
+of any schema, you can use the following rules:
+
+.. literalinclude:: schema-access.json
+    :language: json
+
+.. _table_rules:
+
+Table Rules
+-----------
+
+These rules govern the privileges granted to users on tables. The user is
+granted the privileges based on the first matching rule read from top to
+bottom. If no rule matches, the user is assumed to have no privileges on the
+table. If no table rules are specified, all users are treated as having all
+privileges by default. Each rule is composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``table`` (optional): regex to match against table names. Defaults to ``.*``.
+* ``privileges`` (required): zero or more of ``SELECT``, ``INSERT``,
+  ``DELETE``, ``OWNERSHIP``, ``GRANT_SELECT``
+
+For example, to allow a user ``admin`` all privileges on all tables in all
```

Comment:
Break this sentence up into a list of what happens... something like:

The following rules define the following table access:

* grant all privileges to the user ``admin`` on all tables in all schemas
...

Reply:
Should I make this change to other rules too to be consistent?

Reply:
Lets just do it here only for now